sensor/common/scannerdefinitions/http_handler_test.go

```diff
@@ -7,9 +7,12 @@ import (
 	"net/url"
 	"testing"
 
+	"github.com/stackrox/rox/pkg/httputil"
 	"github.com/stretchr/testify/assert"
 )
 
+var _ http.ResponseWriter = (*responseWriterMock)(nil)
```

Comment:
Why do you have this?

Reply:
This ensures the code will not compile if `*responseWriterMock` no longer implements the `http.ResponseWriter` interface. See https://github.com/uber-go/guide/blob/master/style.md#verify-interface-compliance